### PR TITLE
Keep logger metadata from caller

### PR DESCRIPTION
**Pull request by nvieirafelipe**

### CODE REVIEW

File: lib/meta_logger.ex [hunk 1]
```diff
@@ -1,18 +1,68 @@
...
+  def log(level, chardata_or_fun, metadata \\ []) do
+    merge_logger_metadata_from_parent_processes()
+
+    Logger.log(level, chardata_or_fun, metadata)
+  end
+
+  @spec merge_logger_metadata_from_parent_processes() :: :ok
+  defp merge_logger_metadata_from_parent_processes do
+    "$callers"
+    |> String.to_atom()
```
Comment (prodis) [inline lib/meta_logger.ex]: Why not `:"$callers"` directly?

File: lib/meta_logger.ex [hunk 2]
```diff
@@ -1,18 +1,68 @@
...
+    |> Process.get()
+    |> List.wrap()
+    |> Enum.each(&get_process_logger_metadata/1)
+  end
+
+  @spec get_process_logger_metadata(pid()) :: :ok
+  defp get_process_logger_metadata(process) do
+    case get_in(Process.info(process), @metadata) do
```
Comment (prodis) [inline lib/meta_logger.ex]: Suggestion:

```elixir
process
|> Process.info()
|> get_in(@metadata)
|> case do
...
```

File: mix.exs [hunk 3]
```diff
@@ -6,17 +6,15 @@ defmodule MetaLogger.MixProject do
...
-      docs: [
-        main: "MetaLogger",
-        source_ref: @version,
-        source_url: "https://github.com/FindHotel/meta_logger"
-      ]
+      dialyzer: dialyzer(),
+      docs: docs(),
+      elixir: "~> 1.9",
+      package: package(),
+      source_url: "https://github.com/FindHotel/meta_logger",
```
Comment (prodis) [inline mix.exs]: Github URL can be moved to a module attribute.

File: lib/meta_logger.ex (hunk 1 above)
Comment (nvieirafelipe) [reply lib/meta_logger.ex]: yep :smile:

**Pull request merged**